src/components/CollectionList.vue

```diff
@@ -163,12 +167,41 @@ export default {
 			return options
 		}
 	},
+
 	mounted() {
 		actions.fetchCollectionsByResource({
 			resourceType: this.type,
 			resourceId: this.id
 		})
 	},
+
+	watch: {
```

Comment:
`The "watch" property should be above the "mounted" property on line 171`

Reply:
Thanks, fixed